frontend/bank-transfer-ui/src/app/login/login.component.css

```diff
@@ -28,14 +38,16 @@
   color: rgb(255, 255, 255);
   width: 100%;
   border-radius: 0;
-  border: 2px solid rgb(255, 255, 255);
   padding: 10px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
```

Comment:
Do -webkitów możesz użyć wtyczki, która Ci to ogarnie zależnie od wersji przeglądarek